health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadData.js

```
@@ -1015,142 +1057,200 @@
+            type={"default"}
+            className={"popUpClass"}
+            footerclassName={"popUpFooter"}
+            heading={
+              type === "boundary"
+                ? t("ES_CAMPAIGN_UPLOAD_BOUNDARY_DATA_MODAL_HEADER")
+                : type === "facilityWithBoundary"
+                ? t("ES_CAMPAIGN_UPLOAD_FACILITY_DATA_MODAL_HEADER")
+                : t("ES_CAMPAIGN_UPLOAD_USER_DATA_MODAL_HEADER")
+            }
+            children={[
+              <div>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)